src/hooks/screen/useChatScreenHook.ts

```diff
@@ -87,15 +117,22 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       }
 
       let response;
-      if (type === 'ANONYMOUS') {
-        response = await AnonymousMessageRepo.sendAnonymousMessage(selectedChannel?.id, message);
+      if (type === ANONYMOUS) {
+        response = await AnonymousMessageRepo.sendAnonymousMessage(
+          selectedChannel?.id,
+          message,
+          replyData?.id
+        );
       } else {
         response = await SignedMessageRepo.sendSignedMessage(
           selectedChannel?.id,
           message,
-          channelType
+          channelType,
+          replyData?.id
         );
       }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The sending of messages is updated to include reply data. Make sure that the `sendAnonymousMessage` and `sendSignedMessage` functions properly handle this new parameter.
```diff
+ if (type === ANONYMOUS) {
+         response = await AnonymousMessageRepo.sendAnonymousMessage(
+           selectedChannel?.id,
+           message,
+           replyData?.id
+         );
+        } else {
+          response = await SignedMessageRepo.sendSignedMessage(
+            selectedChannel?.id,
+            message,
+           channelType,
+           replyData?.id
+          );
+        }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->